app/mci/pkg/gcp/healthcheck/healthchecksyncer_test.go

```diff
@@ -37,18 +39,69 @@ func TestEnsureHealthCheck(t *testing.T) {
 	if _, err := hcp.GetHealthCheck(hcName); !utils.IsHTTPErrorCode(err, http.StatusNotFound) {
 		t.Fatalf("expected NotFound error, actual: %v", err)
 	}
+	force := true
 	err := hcs.EnsureHealthCheck(lbName, []sp.ServicePort{
 		{
 			Port:     port,
 			Protocol: "HTTP",
 		},
-	})
+	}, force)
 	if err != nil {
-		t.Fatalf("expected no error in ensuring health check, actual: %v", err)
+		t.Fatalf("expected no error in ensuring health check (force=true), actual: %v", err)
 	}
 	// Verify that GET does not return NotFound.
 	if _, err := hcp.GetHealthCheck(hcName); err != nil {
 		t.Fatalf("expected nil error, actual: %v", err)
 	}
+
+	force = false
+	err = hcs.EnsureHealthCheck(lbName, []sp.ServicePort{
+		{
+			Port:     port,
+			Protocol: "HTTPS", /* a different protocol */
```

Comment:
what does 'a different protocol mean'? only that it's not http?

Reply:
yes...